nailgun/entities.py

```diff
@@ -1888,6 +1890,18 @@ def create_payload(self):
             payload['system_uuids'] = payload.pop('system_ids')
         return payload
 
+    def read(self, entity=None, attrs=None, ignore=None):
+        """Ignore 'host' field as it is not returned by the server.
+
+        For more information, see `Bugzilla #1325989
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1325989>`_.
+
+        """
+        if ignore is None:
```

Comment:
I don't need host attrs here, i'm just passing received attrs to super with no changes. Purpose of this `read()` is to add 1 field to ignore and that's it. Anyway, that's a temporary solution - mentioned BZ should be fixed pretty soon and whole `read()` will be dropped.